.github/workflows/e2e.yml

```diff
@@ -12,7 +12,11 @@ on:
       - ".ci/jenkins/**"
 
 env:
-  GO_VERSION: 1.19
+  GO_VERSION: 1.21
+  # Kubernetes version should aligned with OCP LTS, aka OCP 4.10 (k8s 1.23) for now
+  # https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md
+  # WARNING: kindest/node is not always existing with given version ...
+  KUBERNETES_VERSION: v1.26.3
```

Comment:
I'm not sure this is used anymore. The kubernetes version now is linked to the kind version (currently at v0.20.0 which [uses](https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0) v1.27.3).
What do you think to capture the kind version to download instead? If so, take care of changing the Makefile target to capture it from the environment if not provided. Right now is hardcoded to v0.20.0.